src/components/address-book/EntryDialog/index.tsx

```diff
@@ -38,7 +39,16 @@ const EntryDialog = ({
     defaultValues,
     mode: 'onChange',
   })
-  const { handleSubmit, formState } = methods
+
+  const { handleSubmit, formState, watch, trigger } = methods
+
+  const address = watch('address')
+
+  useEffect(() => {
+    if (address) {
+      trigger('address')
+    }
+  }, [address, trigger])
```

Comment:
I was trying to avoid this use effect but RHF listens only to onChange and onBlur and doesn't seem to provide another way to trigger the validation. See https://react-hook-form.com/api/useform